win_etw_macros/src/lib.rs

```diff
@@ -646,6 +659,7 @@ fn trace_logging_events_core(attr: TokenStream, item_tokens: TokenStream) -> Tok
 
             #[allow(unused_variable)]
             pub const PROVIDER_GUID: ::win_etw_provider::GUID = #provider_guid_const;
+            pub const PROVIDER_NAME: &'static str = #provider_name;
```

Comment:
trivial: you can drop the `'static` from `const` definitions; it's implicit now.

Reply:
I originally didn't have the `'static`, but when the pipeline ran its tests, it was using 1.55.0 and failed the PR. I don't think I have access to update which version of Rust it's using.